docs/modules/ROOT/pages/how-tos/exoscale/decommission.adoc

```diff
@@ -0,0 +1,95 @@
+= Uninstallation on Exoscale
+
+[abstract]
+--
+Steps to remove an OpenShift 4 cluster from https://exoscale.com[Exoscale].
+--
+
+[NOTE]
+--
+- The commands are idempotent and can be retried if any of the steps fail.
+- In the future, this procedure will be mostly automated
+--
+
+== Prerequisites
+
+* Exoscale https://community.exoscale.com/documentation/iam/quick-start/#api-keys[API key]
+* `terraform` >= 0.14
+* `exo` >= v1.14.0, https://community.exoscale.com/documentation/tools/exoscale-command-line-interface[Exoscale CLI]
+
+== Cluster Decommission
+
+. Export the following vars
++
+[source,console]
+----
+export EXOSCALE_ACCOUNT=<exoscale-account>
+export EXOSCALE_API_KEY=<exoscale-key>
+export EXOSCALE_API_SECRET=<exoscale-secret>
+export CLUSTER_ID=<cluster-name>
+export BASE_DOMAIN=<cluster-base-domain>
+export EXOSCALE_REGION=<cluster-region>
```

Comment:
Could also be done more in a batch style, but no hard feeling
```suggestion
export \ 
  EXOSCALE_ACCOUNT=<exoscale-account> \
  EXOSCALE_API_KEY=<exoscale-key> \
  EXOSCALE_API_SECRET=<exoscale-secret> \
  CLUSTER_ID=<cluster-name> \
  BASE_DOMAIN=<cluster-base-domain> \
  EXOSCALE_REGION=<cluster-region>
```

Reply:
I prefer having single exports, as they're easier to selectively fetch from the shell history or edit.